packages/terra-clinical-slide-group/src/SlideGroup.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import CSSTransitionGroup from 'react-transition-group/CSSTransitionGroup';
+
+import Slide from './Slide';
+
+import './SlideGroup.scss';
+
+const propTypes = {
+  /**
+   * The array of components for the group. Only the last component is visible. The others are hidden but still mounted.
+   */
+  items: PropTypes.array,
+  /**
+   * When true, the transition between slides is not animated.
+   */
+  animationIsDisabled: PropTypes.bool,
+};
+
```

Comment:
It was causing weird logic checking for undefined boolean's. I am not sure if this would be an issue anywhere here, but it was thought it was best practice to always have the boolean defined.